pkgs/by-name/sc/scalar/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  makeWrapper,
+  makeDesktopItem,
+  copyDesktopItems,
+  electron,
+  dpkg,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Consider using the `finalAttrs` pattern to remove the use `rec` and improve the flexibility when overriding, see:
- #315337